### PR TITLE
Update vt_scenarios.xml

### PR DESCRIPTION
**Pull request by belphegor-belbel**
"abstract" tag cannot contain "itemizedlist" directly. It should be placed in "para" instead.
https://tdg.docbook.org/tdg/5.0/abstract.html

### PR creator: Description

This patch fixes a build error in daps.

### PR creator: Are there any relevant issues/feature requests?

Nothing.

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [X] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [X] SLE 15 SP4/openSUSE Leap 15.5
  - [ ] SLE 15 SP4/openSUSE Leap 15.4
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by tbazant** [comment]
What this PR suggests is not correct &lt;itemizedlist/> cannot be placed inside &lt;para/>.  Moreover, the `main` branch as it is validates just fine - what is the need for such change at all?

**Comment by belphegor-belbel** [comment]
Oops, I should have explained the actual problem which I'm facing.
Could you please check OBS build log: https://build.opensuse.org/public/build/home:belphegor_belbel:doc/openSUSE_Leap_15.5/x86_64/opensuse-manuals_ja/_log ?

DAPS shows an error as follows:
> [   75s] Fatal error: The document contains XML errors:
> [   75s] /home/abuild/rpmbuild/BUILD/doc-opensuse-ja-15.5.git.1688508825.4f871907/build/.profiled/x86_64_community-project_osuse/vt_scenarios.xml:12:21: error: element itemizedlist not allowed here; expected the element end-tag or element anchor, formalpara, para or simpara
(please note that this build is done for our translation, but it's caused by a problem with the original source)

**Comment by tbazant** [comment]
@belphegor-belbel i've made changes that should validate ok, can you confirm please?

**Comment by belphegor-belbel** [comment]
&lt;/para&gt; tag is seemed to be redundant, another closing tag is before &lt;sect1&gt;.

File: xml/vt_scenarios.xml [hunk 1]
```diff
@@ -15,46 +15,45 @@
...
+              disaster recovery
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              load balancing
+            </para>
+          </listitem>
+        </itemizedlist>
       </para>
```
Comment (tbazant) [inline xml/vt_scenarios.xml:55]:
```suggestion

```

**Comment by tbazant** [comment]
> </para> tag is seemed to be redundant, another closing tag is before <sect1>.

actually, the ending one was redundant :-) now it should be ok

**Pull request merged**